apps/browser-extension/src/css/vars.ts

```diff
@@ -0,0 +1,171 @@
+export const themeValues = {
```

Comment:
Can we import these from the new UI package as modules? Like 
```js
import { themeValues, borderRadius } from '@reown/appkit-ui-new'
```
Maybe we could avoid code duplication?

Reply:
Good idea, now that we have those in the new design system i'll import it from the new package 🙏